templates/zerver/emails/missed_message.subject.txt

```diff
@@ -1,11 +1,11 @@
 {% if show_message_content %}
     {% if group_pm %} Group PMs with {{ huddle_display_name }}
     {% elif mention %} {{ sender_str }} mentioned you
-    {% elif private_message %} {{ sender_str }} sent you a message
-    {% elif stream_email_notify %} New messages in {{ messages[0].header.plain }}
+    {% elif private_message %} PMs with {{ sender_str }}
+    {% elif stream_email_notify %} #{{ messages[0].header.plain }}
```

Comment:
Here I have changed `Zoe sent you a message` to `PMs with Zoe`

and `New messages in Verona > Verona2` to `#Verona > Verona2`